src/controllers/userController.ts

```diff
@@ -19,21 +19,20 @@ interface MongoError {
 export class UserController {
   public async register(req: Express.Request, res: Express.Response) {
     const { username, password } = <RegisterDto>matchedData(req);
-    if (password && username) {
+    if (!password || !username) {
+      return res.status(StatusCodes.BAD_REQUEST).send('Missing username or password');

```

Comment:
you should think on add `error_list` file np in `helpers` folder with all error descriptions like this because it should be const if you want to use it in another place.